app/src/ui/branches/no-pull-requests.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import { Ref } from '../lib/ref'
+import { LinkButton } from '../lib/link-button'
+
+const BlankSlateImage = `file:///${__dirname}/static/empty-no-file-selected.svg`
+
+interface INoPullRequestsProps {
+  /** The name of the repository. */
+  readonly repositoryName: string
+
+  /** Is the default branch currently checked out? */
+  readonly isOnDefaultBranch: boolean
+
+  /* Called when the user wants to create a new branch. */
+  readonly onCreateBranch: () => void
+
+  /** Called when the user wants to create a pull request. */
+  readonly onCreatePullRequest: () => void
+}
+
+/** The placeholder for when there are no open pull requests. */
+export class NoPullRequests extends React.Component<INoPullRequestsProps, {}> {
+  public render() {
+    return (
+      <div className="no-pull-requests">
+        <img src={BlankSlateImage} className="blankslate-image" />
+
+        <div className="title">You're all set!</div>
```

Comment:
Personal pref, but I think text should be wrapped in a `p`

Reply:
I don't think it matters. We use `div` in lots of places.